src/TripSector/TripDate/index.js

```diff
@@ -0,0 +1,37 @@
+// @flow
```

Comment:
`TripDate` occurs once time in `TripSector` right ? better solution is pass `date` as prop

Reply:
`TripDate` is not used in `TripSector` at all.